### PR TITLE
Prepare for new Uno

### PR DESCRIPTION
**Pull request by mortend**
Some build adjustments due to changes in the new Uno version.

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

**Comment by mortend** [comment]
AppVeyor :/

```
Since you ran multiple projects, here is a summary:
From 35 projects, ran 1213 tests, 3 failed.
  Failed:  Fuse.Navigation.Test.RouterTest.RelativeNest
  Failed:  Fuse.Navigation.Test.RouterTest.RelativeNestModify
  Failed:  Fuse.Navigation.Test.ActivatedTest.Deep
```

File: Source/AllTests.unoproj [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "BuildCondition": "UNO_TEST",
```
Comment (kusma) [inline Source/AllTests.unoproj]: Hmm, this makes it sound like uno#197 is a regression, and that similar issues might be triggered in other repos...

File: Source/AllTests.unoproj [hunk 2]
```diff
@@ -1,9 +1,5 @@
 {
   "BuildCondition": "UNO_TEST",
-  "Packages": [
-    "Fuse",
-    "FuseJS",
-  ],
```
Comment (kusma) [inline Source/AllTests.unoproj]: Nice :+1:

**Comment by mortend** [comment]
Yeah, it's a little bit iffy... But will do more good than harm I think...

A better solution might be to introduce project types (Library, Test, App), but that's more work and will also require changing project files in repos. `uno doctor` will only care about Library projects, `uno test` about Test, and so on. We can keep backwards compatibility and give warnings on projects missing a project type. Might implement this later.

**Comment by kusma** [comment]
Yeah, let's see if it turns out to be a problem or not.

File: Source/AllTests.unoproj (hunk 1 above)
Comment (mortend) [reply Source/AllTests.unoproj]: The regression would be that the new version of `uno doctor` will find and build more projects. Shouldn't be a blocker if it happens though.

**Pull request merged**